src/app/interceptors/http.interceptor.ts

```diff
@@ -56,10 +57,15 @@ export class HttpErrorInterceptor implements HttpInterceptor {
      * This can be used to not display error message for certain requests.
      */
     private readonly ignoredEndpoints: IgnoredRequests[] = [
+        { endpoint: 'highlight', method: HttpMethods.GET },
         { endpoint: 'highlight/project/', method: HttpMethods.GET },
         { endpoint: 'wizard', method: HttpMethods.GET },
     ];
 
+    private readonly ignoredStatusCodes: number[] = [
+        404
```

Comment:
I think we discussed this before, but this is only a temporary fix, right?

Reply:
Yes until we get rid of 404 as described in the issue here https://github.com/DigitalExcellence/dex-backend/issues/207

Reply:
Yes this is only temporary